### DISCUSSION TITLE
Preventing unfocused windows from slowing down

### DISCUSSION THREAD

**Discussion by BenMac31** [hyprwm/Hyprland#2059] [General]
When I unfocus a window Eg: a game I am afking in. The window experiences extreme stuttering and fps drops. I'd like to be able to have the window unfocused while still being able to tell what's going on. When I looked through the window rules that could be applied I didn't see anything for this. Is there any way to prevent windows from slowing down when unfocused?

**Comment by vaxerski** [comment]
it's the game doing that, not hyprland. Many games slow down when unfocused to conserve resources. Especially wine games, because in windows, unfocused means minimized for fullscreen games.

**Reply by BenMac31** [reply]
Thanks for telling me this. Is there any way to trick a window into thinking it's still focused when it's not?

**Reply by vaxerski** [reply]
No
